python/tvm/tir/tensor_intrin/hexagon.py

```diff
@@ -64,6 +64,50 @@ def dot_product_32x4_u8u8i32_vrmpy(
         )
 
 
+@T.prim_func
+def dot_product_32x4_u8i8i32_vtcm_desc(
+    A: T.Buffer((4,), "uint8", offset_factor=1, scope="global.vtcm"),
+    B: T.Buffer((32, 4), "int8", offset_factor=1, scope="global.vtcm"),
+    C: T.Buffer((32,), "int32", offset_factor=1, scope="global.vtcm"),
```

Comment:
Is it possible to parametrize `scope` to remove duplication?

Reply:
Yes! ✅